palantirnet/search_api_federated_solr
# Added page title to config form.

[PL-28-Make-title-editable-optional](https://palantir.atlassian.net/browse/PL-28)

*****#REVIEW [THE REACT PR](https://github.com/palantirnet/federated-search-react/pull/12) BEFORE REVIEWING THIS*****

##Description

- Added Page Tile field to federated search config
- Added Logic to react app to display page title when set or hide if it isn't. [React PR](https://github.com/palantirnet/federated-search-react/pull/12)


##Testing

- Spin up a test environment with this branch of the module
- Make sure the react app is up to date with the changes in [this PR](https://github.com/palantirnet/federated-search-react/pull/12) 
- visit /admin/config/search-api-federated-solr/search-app/settings and add a page title.
- View your search page and check that the title is correct.
- visit /admin/config/search-api-federated-solr/search-app/settings and remove the page title.
- View your search page and check that the title and h1 tags are hidden.


##TODO

- [x] Merge [react PR](https://github.com/palantirnet/federated-search-react/pull/12) first.
